docs/GettingStartedDocs/Contributors/building_oe_sdk.md

```diff
@@ -21,7 +21,7 @@ Please refer to the following [documentation](/docs/GettingStartedDocs/SGXSuppor
    - `SGX1+FLC`: In this mode, the Open Enclave SDK takes advantage of the Flexible Launch
                  Control mode for better managing architectural enclaves.
 
-   - `Simulator`: Open Enclave comes with a SGX software simulator that simulates a subset of
+   - `Simulation`: Open Enclave comes with a SGX software simulator that simulates a subset of
```

Comment:
Line 24 and 25 both still have the word "simulator" in them.  I think both should be changed to "simulation mode".

Reply:
Fixed.